js/modules/k6/http/request.go

```diff
@@ -444,7 +475,11 @@ func (h *HTTP) Batch(ctx context.Context, reqsV goja.Value) (goja.Value, error)
 	}
 
 	if err != nil {
-		return nil, err
+		if state.Options.Throw.Bool {
+			return nil, err
+		}
+		state.Logger.WithField("error", err).Warn("A batch request failed")
```

Comment:
Are we fine with this returning a different message than for a single request failure (`Request Failed`)? I thought it would be useful for troubleshooting to know it was a batch request, but maybe this should have the same message for consistency?